src/csvcubed/cli/entrypoint.py

```diff
@@ -125,3 +128,45 @@ def inspect_command(log_level: str, csvw_metadata_json_path: Path) -> None:
         if isinstance(e, HasErrorUrl):
             _logger.error(f"More information available at {e.get_error_url()}")
         sys.exit(1)
+
+
+@entry_point.group("code-list", help="Something something something.")
```

Comment:
Please change this help message.

Reply:
Create a skos-flavoured code list from a JSON file. would this work?

Reply:
> Create a SKOS code list from a JSON file.

Would be what I'd say.